### DISCUSSION TITLE
OSError: [E050] Can't find model 'es_core_news_sm'

### DISCUSSION THREAD

**Discussion by Catalina1100** [explosion/spaCy#8881] [Help: Installation] [answered]
Hi!

I was trying to load the Spanish model in Google Collab and got this error.

**OSError: [E050] Can't find model 'es_core_news_sm'**

Here is the code I used:

```
#!spacy download es_core_news_sm
import spacy

nlp = spacy.load("es_core_news_sm")
doc = nlp("Sevilla es una maravilla.")
for token in doc:
    print(token.text)
```

Does anyone know how I couldfix it, please? 

Cheers,
Katerina

## Your Environment

* Operating System:
* Python Version Used:
* spaCy Version Used:
* Environment Information:

**Comment by munhouiani** [comment, marked as answer]
Did you download the Spanish model before?

`!python -m spacy download es_core_news_sm`

EDIT:

The spacy version on the google colab seems to be `2.2.4` and the problem persists even after I run the installation command. So if you upgrade the spacy to its latest version, it works.

```
! pip install spacy~=3.1
! python -m spacy download es_core_news_sm
```

**Comment by Catalina1100** [comment]
Yes, I did download it. Thanks. Is there anything else I missed?

**Comment by munhouiani** [comment]
You should be able to load the model now
```
nlp = spacy.load("es_core_news_sm")
```

**Comment by polm** [comment]
You can check which spaCy version you have and what models are installed using this command:

    python -m spacy info --markdown

The advice munhouiani gave should work. If you're just getting started with spaCy I would definitely recommend using the latest version rather than an old version.

**Comment by Catalina1100** [comment]
Thanks a bunch!
